src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+
+  --blue-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo__link {
+  line-height: 0;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font for 'Roboto' to ensure text is displayed correctly if the primary font is unavailable. For example: 'font-family: Roboto, Arial, sans-serif;'.